### PR TITLE
Add friends count

### PR DESCRIPTION
**Pull request by geeknees**
Screenshot 2023-10-27 at 14-22-26 ConferenceApp

Screenshot 2023-10-27 at 14-20-52 ConferenceApp

Added count of friends. Please check

### CODE REVIEW

File: app/views/profiles/index.html+mobile.erb [hunk 1]
```diff
@@ -48,7 +48,7 @@
         </button>
       <% end %>
       <div>
-        <h3 class="text-2xl font-bold mb-2">知り合った人達</h3>
+        <h3 class="text-2xl font-bold mb-2">知り合った人達<%= @friends.present? ? " (#{@friends.size})" : '' %></h3>
```
Comment (unasuke) [inline app/views/profiles/index.html+mobile.erb:51]:
```suggestion
        <h3 class="text-2xl font-bold mb-2">知り合った人達<%= @friends.present? ? " (#{@friends.size}人)" : '' %></h3>
```

Comment (unasuke) [reply app/views/profiles/index.html+mobile.erb:51]: @geeknees How do you think about it?

File: app/views/profiles/index.html.erb [hunk 2]
```diff
@@ -45,7 +45,7 @@
       <%= link_to "編集する", edit_profile_path(@profile), type: "button", class: "rounded-md bg-white px-3.5 py-2.5 font-semibold text-gray-900 shadow-sm ring-1 ring-gray-300 hover:bg-gray-50" %>
     </div>
     <div>
-      <h3 class="text-2xl font-bold mb-2">知り合った人達</h3>
+      <h3 class="text-2xl font-bold mb-2">知り合った人達<%= @friends.present? ? " (#{@friends.size})" : '' %></h3>
```
Comment (unasuke) [inline app/views/profiles/index.html.erb:48]:
```suggestion
      <h3 class="text-2xl font-bold mb-2">知り合った人達<%= @friends.present? ? " (#{@friends.size})人" : '' %></h3>
```

🙏

File: app/views/profiles/index.html+mobile.erb (hunk 1 above)
Comment (geeknees) [reply app/views/profiles/index.html+mobile.erb]: Thank you! I like it.

**Review by unasuke — APPROVED** [review]
🥰

**Pull request merged**